src/components/Table/components/AgentSection.vue

```diff
@@ -0,0 +1,28 @@
+<script setup>
+defineProps({
+  name: String,
+  email: String,
+  thumbnail: String,
+});
+</script>
+<template>
```

Comment:
I would rename this file something more generic like UserCard or something of that sort.